server/webapp/WEB-INF/rails.new/app/assets/javascripts/console_log_socket.js

```diff
@@ -64,6 +64,13 @@
     function maybeResume(e) {
       if (fatal) return;
 
+      if (e.type === "close" && e.code === 1011) {
+        startLine = 0;
+
+        console.error('Something went wrong:'+ e.reason);
+        return;
+      }
```

Comment:
@ketan -- Merging this for now.
This can be fixed as a part of **Custom WebSocket Wrapper** that we have written which falls back to polling and has a better error handling!